### PR TITLE
[DO NOT MERGE] Rate limit the vulnerable people landing page

### PR DESCRIPTION
**Pull request by maxgds**
This branch is based on prior art found here: https://github.com/alphagov/frontend/pull/974/files

It piggy backs on multivariant testing code to deliver a rate limit warning to users asking them to refresh later. 

To view, go to /coronavirus-extremely-vulnerable and reload repeatedly until the alternative page is served:
Screenshot 2020-03-23 at 09 08 08

- [x] Copy updated.

- [ ] New cookie is being added to the required list.

### CODE REVIEW

File: app/views/content_items/answer.html.erb [hunk 1]
```diff
@@ -4,4 +4,13 @@
   ) %>
 <% end %>
 
+<% if params[:path] && params[:path] == 'coronavirus-extremely-vulnerable' %>
+  <div id="answer-b-content" class="hidden">
+      <%= render "govuk_publishing_components/components/error_alert", {
+  message: "The register for support service is working but only allowing a certain number of people to register at once. Please wait 5 minutes and refresh this page to try again."
```
Comment (alex-ju) [inline app/views/content_items/answer.html.erb]: Stephen, our Content Designer suggests: `This service is working, but we're limiting the number of people who are using it at the same time. Please wait 5 minutes and refresh the page to try again.`

File: app/assets/javascripts/coronavirus-extremely-vulnerable.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        );
+      }
+    };
+
+    var bContent = $('#answer-b-content').html();
+    var pcThreshold = 20;
+
+    new GOVUK.MultivariateTest({
+      el: '.answer',
+      name: 'coronavirusExtremelyVulnerable_rateLimit_202003_' + pcThreshold.toString() + 'pc_' + minutesForCohort.toString() + 'm',
```
Comment (theseanything) [inline app/assets/javascripts/coronavirus-extremely-vulnerable.js]:
```suggestion
      name: 'coronavirusExtremelyVulnerableRateLimit'
```

**Comment by timblair** [comment]
Can we close this now in favour of alphagov/frontend#2298?

**Comment by alex-ju** [comment]
@timblair sure.

**Pull request closed without merging**